garage/tf/models/gaussian_mlp_model.py

```diff
@@ -108,19 +126,26 @@ def _build(self, state_input):
                     np.zeros(action_dim),
                     np.full(action_dim, self._init_std_param)
                 ], axis=0)  # yapf: disable
-                b = tf.constant_initializer(b)
+
+                def p():
+                    return tf.constant_initializer(b)
```

Comment:
just use a lambda

Reply:
I used lambda in the first place and Flake8 E731 suggested not to assign a lambda function. Should we disable E731 in flake8?

Reply:
i don't think it should complain if you assign the lambda as a kwarg